enhancements/windows-containers/windows-machine-config-operator.md

```diff
@@ -96,15 +96,15 @@ will have all the pre-requisite steps needed for setting the VM up for key based
 SSH connections.
 
 When the cluster admin wants to bring up Windows worker nodes, they will create
-a [MachineSet](https://github.com/openshift/machine-api-operator/blob/master/config/machineset.yaml)
-where they reference a Windows OS image which has the Docker container runtime
```

Comment:
You again need to retain this bullet and reference the WMCO versions where we support Docker.